storage/mysql/storage_test.go

```diff
@@ -256,36 +258,20 @@ func createFakeSignedLogRoot(db *sql.DB, tree *trillian.Tree, treeSize uint64) {
 		return nil
 	})
 	if err != nil {
-		panic(fmt.Sprintf("ReadWriteTransaction() = %v", err))
+		t.Fatalf("ReadWriteTransaction() = %v", err)
 	}
 }
 
 // createTree creates the specified tree using AdminStorage.
-func createTree(db *sql.DB, tree *trillian.Tree) (*trillian.Tree, error) {
-	ctx := context.Background()
-	s := NewAdminStorage(db)
+func createTree(ctx context.Context, t *testing.T, s storage.AdminStorage, tree *trillian.Tree) *trillian.Tree {
```

Comment:
can you rename this to something like `mustCreateTree` or `createTreeOrDie` to make it obvious it can exit.